### ISSUE TITLE
Is there any documentation

### ISSUE THREAD

**Issue by rookieLink** [dagrejs/dagre-d3#379]
I want some document about dagre-d3, is there any documention that it can show me how to use and  what the method to do ?

**Comment by ryw** [comment]
Docs are here https://github.com/dagrejs/dagre-d3/wiki

**Comment by johanwasserman** [comment]
Docs @ https://github.com/dagrejs/dagre-d3/wiki are seriously lacking looking at the source and capabilities.

The difference between a so-so tool and a GREAT tool is documentation.  Would be great to have a full API doc to work with instead of spending hours scanning source code for what the tool can and can't do.

**Comment by rookieLink** [comment]
@johanwasserman Yes, this is what I want to say. We need a good document to help us understand and use, instead of spending a lot of time to scan source

**Comment by corwin-of-amber** [comment]
There are many options on graph, node, and edge elements. I find them anecdotally in scattered examples across the Web. It would be sooo good to have a comprehensive list.

**Comment by gordonwoodhull** [comment]
I think at this point it would have to be a community effort, since the original author left some time ago.
